Detectors/FIT/FT0/simulation/src/digit2raw.cxx

```diff
@@ -48,7 +48,8 @@ int main(int argc, char** argv)
     add_option("verbosity,v", bpo::value<int>()->default_value(0), "verbosity level");
     //    add_option("input-file,i", bpo::value<std::string>()->default_value(o2::base::NameConf::getDigitsFileName(o2::detectors::DetID::FT0)),"input FT0 digits file"); // why not used?
     add_option("input-file,i", bpo::value<std::string>()->default_value("ft0digits.root"), "input FT0 digits file");
-    add_option("file-per-link,l", bpo::value<bool>()->default_value(false)->implicit_value(true), "create output file per CRU (default: per layer)");
+    add_option("flp-name", bpo::value<std::string>()->default_value("alio2-cr1-flp180"), "single file per: all,flp,cru,link");
```

Comment:
Actually, the comment for this option seems to be wrong, I guess copy&paste error.

Reply:
yes, sorry, i can change it in next PR(in few hours)

Reply:
oh, I meant after merging this one :) It will be not in few hours